main.rb

```diff
@@ -0,0 +1,103 @@
+require_relative 'app'
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+
+def display_menu
+  puts 'What would you like to do?'
+  puts '1. List all books'
+  puts '2. List all people'
+  puts '3. Create a person'
+  puts '4. Create a book'
+  puts '5. Create a rental'
+  puts '6. List all rentals for a given person id'
+  puts '0. Quit'
+end
+
+def list_all_books(app)
+  app.list_books.each do |book|
+    puts "#{book.title} by #{book.author}"
+  end
+end
+
+def list_all_people(app)
+  app.list_people.each do |person|
+    puts "#{person.name} (#{person.class})"
+  end
+end
+
+def create_person(app)
+  puts "Enter the person's name:"
+  name = gets.chomp
+  puts "Enter the person's age:"
+  age = gets.chomp.to_i
+  puts "Enter the person's type (student or teacher):"
+  type = gets.chomp.downcase
```

Comment:
- [ ] In the option 3, we should see the below prompt, kindly fix this: 
>
![image](https://user-images.githubusercontent.com/91486791/232098503-5fe054cb-733e-4a79-9d6c-de7c57b27f63.png)
> your app: 
![image](https://user-images.githubusercontent.com/91486791/232098592-5b30fbf2-2ce7-4437-82ef-122b23836dbb.png)

Reply:
Done, thank you